<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plant_monitor</class>
 <widget class="QMainWindow" name="plant_monitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>plant_monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget
{
  border-image:url(:/picture/plant_back.png);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_7" stretch="3,9,3">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6" stretch="2,5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,11">
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <family>黑体</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(225, 225, 225);
border:2px groove gray;
border-radius:10px;padding:2px 4px;
border-style: outset;

</string>
                </property>
                <property name="text">
                 <string>IP地址</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="IP_Input">
                <property name="font">
                 <font>
                  <family>黑体</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>192.168.137.32</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <family>黑体</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(225, 225, 225);
border:2px groove gray;
border-radius:10px;padding:2px 4px;
border-style: outset;
</string>
                </property>
                <property name="text">
                 <string>端口号</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="Port_Input">
                <property name="font">
                 <font>
                  <family>黑体</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>8080</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="14,5,5">
            <item>
             <widget class="QLabel" name="WIFI">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>WIFI未连接</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="connect_btn">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(225, 225, 225);
border:2px groove gray;
border-radius:10px;padding:2px 4px;
border-style: outset;

</string>
              </property>
              <property name="text">
               <string>连接</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,2,1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1,5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2">
              <item>
               <widget class="QLabel" name="temperature">
                <property name="font">
                 <font>
                  <family>黑体</family>
                  <pointsize>23</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:rgb(255,99,71);</string>
                </property>
                <property name="text">
                 <string>土壤温度</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="temperature_digit">
                <property name="font">
                 <font>
                  <family>黑体</family>
                  <pointsize>43</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:rgb(255,99,71);</string>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2">
              <item>
               <widget class="QLabel" name="humidity">
                <property name="font">
                 <font>
                  <family>黑体</family>
                  <pointsize>23</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:rgb(0,191,255);</string>
                </property>
                <property name="text">
                 <string>土壤湿度</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="humidity_digit">
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>43</pointsize>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:rgb(0,191,255);</string>
                </property>
                <property name="text">
                 <string>0%</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QLabel" name="light">
                <property name="font">
                 <font>
                  <family>黑体</family>
                  <pointsize>23</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:rgb(255,240,245);</string>
                </property>
                <property name="text">
                 <string>光照强度</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="light_digit">
                <property name="font">
                 <font>
                  <family>黑体</family>
                  <pointsize>43</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:rgb(255,240,245);</string>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="state">
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(255,215,0);</string>
            </property>
            <property name="text">
             <string>状态:未知</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>447</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
